universal-login-ops/src/dev/startDevelopment.ts

```diff
@@ -38,10 +38,17 @@ function getRelayerConfig(jsonRpcUrl: string, wallet: Wallet, walletMasterAddres
       ensAddress,
       chainId: 0
     },
-    ensRegistrars
+    ensRegistrars,
+    validCodeHashes
   };
 }
 
+function getHashDeployedProxyAbicode() {
```

Comment:
Come on! Is that a good name? 'getProxyBytecode()'

Reply:
✔️